common/include/pcl/common/impl/common.hpp

```diff
@@ -70,6 +70,14 @@ pcl::getAngle3D (const Eigen::Vector3f &v1, const Eigen::Vector3f &v2, const boo
 inline void
 pcl::getMeanStd (const std::vector<float> &values, double &mean, double &stddev)
 {
+  // throw an exception when there are less than two elements in vector `values` 
+  // to prevent divide by zero error. Then, let the program crash.
+  // Let the user explicity handle this exception as per his needs.
+  if(values.size() < 2 )
+  {
+    throw std::length_error("Input array must have atleast 2 elements.");
```

Comment:
I'll update this, once you settle on which exception to use.

Reply:
Well there is `BadArgumentException` which sort of fits, we may use it. Though I feel there is no consistency within PCL regarding exceptions.

Reply:
```cpp
throw pcl::BadArgumentException("Input array must have at least 2 elements.", NULL, "getMeanStd");
```
is this alright?

Reply:
I'd use `PCL_THROW_EXCEPTION` following the example here:

https://github.com/PointCloudLibrary/pcl/blob/bf61b8820b27d401ce73a16a5d9952e27a48e72b/common/include/pcl/exceptions.h#L45-L49

Reply:
```cpp
PCL_THROW_EXCEPTION(BadArgumentException, "Input array must have at least 2 elements "); 
```
This will do? I thought mentioning the filename will barely make any sense here, as it's just a utility function.

Reply:
Yes. But please not my other comment regarding the minimum size of the array.

Reply:
Yes. I commented on that thread. Please see to it.